src/app/auth/services/user.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserService {
+  constructor(private http: HttpClient) {}
+
+  addUser(data: Object) {
```

Comment:
Remove typecasting `Object` we dont use this typecast.
Either provide proper interface or use `any`